src/app/static/src/app/apiService.ts

```diff
@@ -7,26 +7,14 @@ import {
     PjnzResponse,
     PopulationResponse,
     ProgrammeResponse,
-    Response,
+    Response, SessionFile,
     ShapeResponse,
-    SurveyResponse,
+    SurveyResponse, ValidateBaselineResponse,
     ValidateInputResponse
 } from "./generated";
 import {Commit} from "vuex";
 import {freezer} from "./utils";
-
-type ResponseData =
```

Comment:
This was pointless and incorrect, since we get all sorts of response back from the API.